src/simplexinv.jl

```diff
@@ -1,57 +1,62 @@
 export simplexinv
 
-function simplexinv(c, A, b, IB=0, invB=0; max_iter = 4000)
+function simplexinv(c, A, b, 𝔹=0, invB=0; max_iter = 20000)

```

Comment:
`B = 0` may lead to a type stability issue. Since `B` should be an `Array{Int,1}`, we could use `B = Int[]` as comparison.